src/blogger.ts

```diff
@@ -0,0 +1,32 @@
+import {HashnodeArticle, DevArticle} from './type';
+import Axios from 'axios';
+
+
+export class Blogger {
+    private readonly dev_token?: string;
+    private readonly hashnode_token?: string;
+
+    constructor(token: {dev?: string, hashnode?: string}) {
+        this.dev_token = token.dev;
+        this.hashnode_token = token.hashnode;
+    }
+
+    postTo = {
+        dev: async (article: DevArticle) => {
+            if(!this.dev_token) return;
+
+            const response = await Axios.post('', article, {
+                headers: {
+                    'Content-Type': 'application/json',
+                    'api-key': this.dev_token
+                }
+            });
+
+        },
```

Comment:
So this is where we will call dev.to API, yeah I am missing the URL still wip, but I don't think there is any way to write tests for it, I have doubt that if I am doing the right thing with the API.